coalahtml/_coalahtml/app/controllers/logs.js

```diff
@@ -9,5 +9,5 @@ angular.module('coalaHtmlApp')
         $scope.data = response.logs;
       });
     };
-    parseCoalaProject("tests/test_projects/simple");
+    parseCoalaProject("data");
```

Comment:
Ok, I got it, why the change, but why it have to be hardcoded I don't understand. I like much more when configuration is separated from the code itself as much as possible and I don't think it's good idea to use somewhat magic constants. At least making a constant variable would make me feel a little better.